test/unit/helpers/mocks/Util.js

```diff
@@ -91,18 +91,22 @@ function ($, _, Backbone, Q, Duo) {
       }
 
       var deferred = $.Deferred();
-      if (xhr.status > 0 && xhr.status < 300) {
-        // $.ajax send (data, textStatus, jqXHR) on success
+
+      (function (textOnly) {
```

Comment:
The issue was a possibility to create the factor.verify() ajax request while one is already running.
Without making mocks for ajax asynchronous for the error results, not only for success, it was impossible to test such situation.